prime-dotnet-webapi/Migrations/20240312233918_AddPharmanetTransactionLogTemp.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class AddPharmanetTransactionLogTemp : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+
+        }
```

Comment:
Shouldn't there be something in the `Up` method?

Reply:
LOL, what am i doing? it is fixed. But the database is not up for this PR...hm